app/pages/about/team-page.cjsx

```diff
@@ -320,7 +317,14 @@ teamMembers =
     title: counterpart "team.content.beverleyNewing.title"
     bio: counterpart "team.content.beverleyNewing.bio"
     image: "/assets/team/beverley.jpg"
-    location: "oxford"
+    location: "alumni"
+beckyRother:
```

Comment:
The indent on this one is off. Like I mentioned, coffeescript doesn't like misaligned indents. This needs to be one indents (2 spaces).